perllib/FixMyStreet/DB/Result/Body.pm

```diff
@@ -231,6 +237,9 @@ sub cobrand_name {
     # If the current body is TfL then we always want to show TfL as the cobrand name.
     return $self->name if $self->name eq 'TfL' || $self->name eq 'National Highways' || $self->name eq 'Environment Agency';
```

Comment:
Given the new code, is this section needed any more? This confusion shouldn't be able to arise now.

Reply:
@davea Just to note I don't think this was removed.